gql/transport/requests.py

```diff
@@ -56,6 +58,7 @@ def __init__(
         self.use_json = use_json
         self.default_timeout = timeout
         self.verify = verify
+        self.method = method
```

Comment:
Should be good to put a little note on the readme about this. I think I will update the readme after merging this.

Reply:
Yep, maybe I let you do it, because I don't see a clear place for documentation arguments. (I could add it to the third example with a comment?)